test/parallel/test-eventtarget.js

```diff
@@ -115,6 +128,43 @@ ok(EventTarget);
   const fn = common.mustCall((event) => strictEqual(event, ev));
   eventTarget.addEventListener('foo', fn, { once: true });
   eventTarget.dispatchEvent(ev);
+  const eventTarget = new EventTarget();
```

Comment:
@benjamingr ... this is failing linting

Reply:
Where? I just ran this locally and didn't get a lint error?